build.gradle

```
@@ -106,9 +105,53 @@ task delombok(type: io.franzbecker.gradle.lombok.task.DelombokTask, dependsOn: c
     }
 }
 
+ext {
+    snippetsDir = file('build/generated-snippets')
```

```suggestion
    snippetsDir = file 'build/generated-snippets'
```
use all the groovy features you can!